keyboards/ogurec/config.h

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2021 drhigsby
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define MANUFACTURER DrHigsby
+#define PRODUCT ogurec
+#define VENDOR_ID 0x0420
```

Comment:
This Vendor ID is in use. 
https://www.the-sz.com/products/usbid/index.php?v=0x0420

Please select a different vendor ID.  Ideally, anything above 0x4000 isn't in use.

Reply:
I submitted this same vendor ID for dubba175 and it was accepted back on March 4th. 

https://github.com/qmk/qmk_firmware/blob/master/keyboards/dubba175/config.h

Should I change the vendor ID on dubba175 as well here for ogurec?

Reply:
changed VID to 0x0007 (which produces no results at www.the-sz.com/products/usbid